Description: ignore the .pc directory when checking for translatable strings.
Author: Kees Cook <kees@ubuntu.com>
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=36415

Index: policykit-1-0.101/po/POTFILES.skip
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ policykit-1-0.101/po/POTFILES.skip	2011-04-19 12:35:33.398977436 -0700
@@ -0,0 +1 @@
+.pc
